database-plugins/src/main/java/io/cdap/plugin/db/batch/sink/DBSink.java

```diff
@@ -90,7 +91,8 @@ private String getJDBCPluginId() {
   @Override
   public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
     super.configurePipeline(pipelineConfigurer);
```

Comment:
should handle exceptions from base class method

Reply:
I am not sure I follow this, @CuriousVini. Are you suggesting that we should handle uncaught exceptions or should we call collector.getOrThrowException here?

Reply:
Or we can modify `ReferenceBatchSink` to use failure collector and collect reference name validation failures as well. That base class will not throw any exceptions